app/Gists/GistClient.php

```diff
@@ -84,6 +85,8 @@ public function starGist($gistId)
         }
 
         if (Auth::check()) {
+            Cache::forget("App\Gists\GistClient::starCount::{$gistId}");
```

Comment:
Hmm... this is an ugly side of using `__METHOD__`... this is pretty gross to call. Let me think about how I'd like to fix that.

Reply:
Maybe for now could we just change these to:

```suggestion
            Cache::forget(self::class . "::starCount::{$gistId}");
```

Still ugly, but a BIT less ugly